### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites- Edit_Screen Reader: Narrator focus is not in proper order after Local IIS link

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#167]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0 
Screen Reader: Narrator
**MAS ID Violated:** MAS1.3.2, MAS2.4.3

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till "Web Sites" in the left menu section and Press Enter.
5) Tab till "More" commands button of the existing website inside the grid and Press Enter.
6) Tab till "Edit" in the expanded menu and Press Enter.
7) Validate that focus order is as per flow of UI design in meaningful and logical sequence.

**Actual Result:**
Narrator focus is not proper after Local IIS link.

**Expected Result:**
Focus order should be as per flow of UI design in meaningful and logical sequence. After Local IIS link focus should land on Web Server and Web Sites link then on Actions button

**User Impact:** 
Visually Impaired Users may face difficulty in navigating through the controls if the focus order for the page is not proper 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachments:**
[Bug_167_Focus order is not proper after Local IIS link.pptx](https://github.com/Microsoft/IIS.WebManager/files/2331639/Bug_167_Focus.order.is.not.proper.after.Local.IIS.link.pptx)

**Comment by jhkimnew** [comment]
Fixed. The current tab navigation between tab and main pane requires some more work in the future to support navigating tab with up/down arrow key.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.2; #C+E-H; #.NETWebsites; #Edge; #Narrator; #MAS2.4.3; #Keyboard

**Comment by Yashi1201** [comment]
**Github tags:** #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
